features/contact/requesting_contact_validation.feature

```diff
@@ -0,0 +1,44 @@
+@requesting_contact
+Feature: Requesting contact validation
+    In order to avoid making mistakes when requesting contact
+    As a Customer
+    I want to be prevented from making mistakes in my email address
```

Comment:
`in my request`?  You check not only if email is correct but also if the message is correct.